scripts/build-example.sh

```diff
@@ -2,8 +2,13 @@
 
 set -e # exit when error
 
-yarn
-yarn build
```

Comment:
this could be removed completely I think, since the `dist` folder doesn't exist if you don't build first

Reply:
I've left it [there](https://github.com/algolia/angular-instantsearch/pull/597/files/33dc08672561b25f4cc4abbaa3bf454499ecb823#diff-c9a5caf39341075848b39fe599785cf8R9), because without it we cannot build a specific example, for example,

```
git clone ...
yarn && yarn examples:media:build
```